python/gui/MaaS/templates/maas/dataset_management.html

```diff
@@ -0,0 +1,797 @@
+                    console.log(response);
+                }
+            });
+        }
+
+        function createFormSelectChange(formInput, selection) {
+            let component = new CreateDatasetForm('dataset-create');
+            component.dynamicInputUpdate(formInput, selection);
+        }
+
+        /*
+        {% for model, model_parameters in parameters.items %}
+        {% for parameter in model_parameters %}
+        function toggle_{{ model }}_{{ parameter.value }}() {
+            var selector = "#{{ model }}_{{ parameter.value }}_scalar_row, ";
+            selector = selector + "#{{ model }}_{{ parameter.value }}_distribution_row";
+
+            $(selector).toggle();
+        }
+
+        function enable_{{ model }}_{{ parameter.value }}_scalar() {
+            var scalarSelector = "#{{ model }}_{{ parameter.value }}_scalar";
+
+            var distributionSelector = "#{{ model }}_{{ parameter.value }}_distribution_min, ";
+            distributionSelector = distributionSelector + "#{{ model }}_{{ parameter.value }}_distribution_max, ";
+            distributionSelector = distributionSelector + "#{{ model }}_{{ parameter.value }}_distribution_type";
+
+            var distributionRowsSelector = "#{{ model }}_{{ parameter.value }}_distribution_type_row, ";
+            distributionRowsSelector = distributionRowsSelector + "#{{ model }}_{{ parameter.value }}_distribution_min_row, ";
+            distributionRowsSelector = distributionRowsSelector + "#{{ model }}_{{ parameter.value }}_distribution_max_row";
+
+            $(distributionSelector).attr('disabled', true);
+            $(distributionRowsSelector).addClass('disabled');
+            $(distributionRowsSelector).hide();
+
+            $(scalarSelector).removeAttr('disabled');
+        }
+
+        function enable_{{ model }}_{{ parameter.value }}_distribution() {
+            var scalarSelector = "#{{ model }}_{{ parameter.value }}_scalar";
+
+            var distributionSelector = "#{{ model }}_{{ parameter.value }}_distribution_min, ";
+            distributionSelector = distributionSelector + "#{{ model }}_{{ parameter.value }}_distribution_max, ";
+            distributionSelector = distributionSelector + "#{{ model }}_{{ parameter.value }}_distribution_type";
+
+            var distributionRowsSelector = "#{{ model }}_{{ parameter.value }}_distribution_type_row, ";
+            distributionRowsSelector = distributionRowsSelector + "#{{ model }}_{{ parameter.value }}_distribution_min_row, ";
+            distributionRowsSelector = distributionRowsSelector + "#{{ model }}_{{ parameter.value }}_distribution_max_row";
+
+            $(scalarSelector).attr('disabled', true);
+
+            $(distributionSelector).removeAttr('disabled');
+            $(distributionRowsSelector).removeClass('disabled');
+            $(distributionRowsSelector).show();
+        }
+        {% endfor %}
+        {% endfor %}
+        */
+    </script>
+    <style>
+        #header {
```

Comment:
We should probably consolidate these and styles from other templates into a common css document to help maintain consistency between pages. Branding, for instance, could be used on just about every page, for example.